### PR TITLE
move ops.point_cloud => point_cloud namespace

### PR DESCRIPTION
**Pull request by LukeWood**

### CODE REVIEW

**Comment by LukeWood** [comment]
ok @ianstenbit I think this should be ready

**Review by ianstenbit — APPROVED** [review]
This LGTM but please do not merge until we've verified that this won't cause downstream breakages.

I'll add Zhaoqi to take a look.

**Comment by ianstenbit** [comment]
Also @LukeWood these test failures are real.

File: keras_cv/point_cloud/point_cloud_test.py [hunk 1]
```diff
@@ -205,15 +203,15 @@ def test_within_box3d(self):
         assert bboxes.shape[0] == num_boxes
         assert expected_is_inside.shape[0] == num_points
         assert expected_is_inside.shape[1] == num_boxes
-        is_inside = ops.is_within_box3d(points, bboxes)
+        is_inside = point_cloud.is_within_box2d(points, bboxes)
```
Comment (ianstenbit) [inline keras_cv/point_cloud/point_cloud_test.py:206]: This is causing the CI failure. This should be `is_within_box3d`

File: keras_cv/point_cloud/point_cloud_test.py [hunk 2]
```diff
@@ -205,15 +203,15 @@ def test_within_box3d(self):
...
         self.assertAllEqual([num_points, num_boxes], is_inside.shape)
         self.assertAllEqual(expected_is_inside, is_inside)
         # Add a batch dimension to the data and see that it still works
         # as expected.
         batch_size = 3
         points = tf.tile(points[tf.newaxis, ...], [batch_size, 1, 1])
         bboxes = tf.tile(bboxes[tf.newaxis, ...], [batch_size, 1, 1])
-        is_inside = ops.is_within_box3d(points, bboxes)
+        is_inside = point_cloud.is_within_box2d(points, bboxes)
```
Comment (ianstenbit) [inline keras_cv/point_cloud/point_cloud_test.py:214]: (same here)

File: keras_cv/point_cloud/point_cloud_test.py (hunk 1 above)
Comment (LukeWood) [reply keras_cv/point_cloud/point_cloud_test.py]: Ahhhh - thanks a lot

File: keras_cv/point_cloud/point_cloud_test.py (hunk 2 above)
Comment (LukeWood) [reply keras_cv/point_cloud/point_cloud_test.py]: Ahhhh - thanks a lot

**Review by lengzq — APPROVED** [review]

**Pull request merged**